src/hoc/withTheme.js

```diff
@@ -89,11 +81,9 @@ const wrapWithTheme = (Component: ComponentType<any>) => {
         delete props.innerRef
       }
 
-      return <Component {...props} />
+      return React.createElement(Component, props)
     }
```

Comment:
This should already transpile to no spread in loose mode :) but it’s always good to be explicit 👌